### PR TITLE
Add ColorTemperature augmentation

### PR DESCRIPTION
**Pull request by r-k-jonynas**

### CODE REVIEW

File: tests/augmentations/test_color_temperature.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    # Warming filters make reds more intensive and blue less.
+    assert np.less_equal(
+        img[:,:,0], aug_img[:,:,0]
+    ).all(), "Performing augmentation does not yield expected red channel"
+    assert np.greater_equal(
+        img[:,:,2], aug_img[:,:,2]
+    ).all(), "Performing augmentation does not yield expected blue channel"
+    assert np.array_equal(
+        bboxes, aug_bboxes
+    ), "Performing augmentation does not yield original augmentation"
```
Comment (ian-datature) [inline tests/augmentations/test_color_temperature.py]: The assert message should state that you are checking the bounding boxes
```suggestion
    ), "Performing augmentation does not yield original bounding boxes"
```

File: tests/augmentations/test_color_temperature.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    # Cooling filters make blues more intensive and reds less.
+    assert np.greater_equal(
+        img[:,:,0], aug_img[:,:,0]
+    ).all(), "Performing augmentation does not yield expected red channel"
+    assert np.less_equal(
+        img[:,:,2], aug_img[:,:,2]
+    ).all(), "Performing augmentation does not yield expected blue channel"
+    assert np.array_equal(
+        bboxes, aug_bboxes
+    ), "Performing augmentation does not yield original augmentation"
```
Comment (ian-datature) [inline tests/augmentations/test_color_temperature.py]: Same as line 30

**Review by ian-datature — CHANGES_REQUESTED** [review]
Please fix the docstring style errors as reported by the linter and we should be go to go. Thanks!

**Review by ian-datature — APPROVED** [review]

**Review by gkeechin — APPROVED** [review]
lgtm

**Pull request merged**